vnext/Shared/Shared.vcxitems

```diff
@@ -14,13 +15,97 @@
+    <ClInclude Include="$(MSBuildThisFileDirectory)DevServerHelper.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)DevSettings.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)etw\react_native_windows.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)HermesRuntimeHolder.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IDevSupportManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IHttpResource.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)INativeUIManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)InstanceManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IReactRootView.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IRedBoxHandler.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IUIManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)IWebSocketResource.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)JSBigAbiString.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)LayoutAnimation.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Logging.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)MemoryMappedBuffer.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)MemoryTracker.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\ExceptionsManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\I18nModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\PlatformConstantsModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\SourceCodeModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\StatusBarManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\UIManagerModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Modules\WebSocketModule.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)NativeModuleProvider.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)OInstance.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Pch\pch.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)ShadowNode.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)ShadowNodeRegistry.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)targetver.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Tracing.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)tracing\fbsystrace.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)TurboModuleManager.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)TurboModuleRegistry.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)Utils.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)V8JSIRuntimeHolder.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)WebSocketJSExecutorFactory.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)WinRTWebSocketResource.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)etw\build.bat" />
```

Comment:
who is using this? are we plopping the ETW stuff into production binary today? 
We might want to make this a build task

Reply:
That's a great question.
I only moved the files and added them to the solution for viewing.
I don't think they're being used by anyone.
Also, there seems to be an actual DLL binary in those sources.

Reply:
looks like it was added in this PR: #3745
There is mention of tools to process the systrace (through JS HUD?) and ETW events (WPA) but I don't see any docs on how to do so.
@mganandraj could you please write something down into an md file? Also, does this DLL need to be checked in? who is using it?

Reply:
Hi .. The resource dll has the schema for the ETW events payload.. Developers can register this resource file against the ETW provider so that teh tools like WPA can parse the ETW events payload.. There is a batch file checked in along with the resource dll to build and register it on a developer machine...  

The resource dll is not supposed to be shipped to the user machine .. It is checked in to make the ETW analysis easier even on a machine with only Windows Perf Toolkit installed but not the build toolchains ..

Reply:
@mganandraj  Can't we consume that DLL as a NuGet dependency?

We shoud NOT push binaries into a Git repository.

Reply:
I don't have any strong opinion here.. This is not a code dll, and it rarely changes and hence there is not overhead of version tracking .. Unfortunately, I'm quite busy with some other tasks for this quarter hence won't be able to get to any cleanups here..